### PR TITLE
[#1011] subjects.csv Changes

### LINKED ISSUE #1011: GF Subjects Requested Changes
Send to @kglibrarian and @gneidhardt

### PR DESCRIPTION
**Pull request by sharpattack**
Add new columns 'isPartOf' and 'id'. closes #1011

### CODE REVIEW

**Review by Meowcenary — APPROVED** [review]
Looks good to me. The larger file has some useful stuff in it for understanding Solr.

File: lib/batch/subject.csv.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 csv = CSV.open('/home/deploy/all_gf_subject.csv', 'wb')
-terms =  GalterGenericFilePresenter.terms + [:depositor, :date_uploaded, :date_modified, :read_access_group]
+terms =  GalterGenericFilePresenter.terms + [
+  :depositor, :date_uploaded, :date_modified, :read_access_group, :isPartOf, :id
+]
 
-csv << terms.map(&:to_s).map { |term| 
-  I18n.translate(:simple_form)[:labels][:generic_file][term] || term.to_s.titleize 
+csv << terms.map(&:to_s).map { |term|
+  I18n.translate(:simple_form)[:labels][:generic_file][term] || term.to_s.titleize
```
Comment (fenekku) [inline lib/batch/subject.csv.rb:7]: Is `term.to_s` still necessary on this line, given it was done on the line above?

**Review by fenekku — APPROVED** [review]
Thanks for this. Can be merged right away after minor comments are addressed.

**Review by gneidhardt — APPROVED** [review]
I do request revisiting "last editor" - it saves me a ton of time if I can see that I was the last editor since then I know that item was reviewed in our previous round of metadata enhancement, but understand it's not possible at this time.

File: lib/batch/subject.csv.rb (hunk 1 above)
Comment (sharpattack) [reply lib/batch/subject.csv.rb]: Nice catch, removed.

**Comment by sharpattack** [comment]
@gneidhardt okay, I can see it is high value now. I added a ticket for it, #1048

**Pull request merged**